tests/src/test/scala/runtime/actionContainers/NodeJsActionContainerTests.scala

```diff
@@ -547,4 +547,42 @@ abstract class NodeJsActionContainerTests extends BasicActionRunnerTests with Ws
         e shouldBe empty
     })
   }
+
+  it should "allow running activations concurrently" in {
+    val (out, err) = withNodeJsContainer { c =>
+      //this action will create a log entry, and only complete once all 3 activations have arrived and emitted logs
+      val code =
+        s"""
+           | function main(args) {
+           |     console.log("interleave me");
+           |     return new Promise(function(resolve, reject) {
+           |         setTimeout(function() {
+           |             resolve({ args: args});
+           |         }, 10000);
+           |    });
+           | }
+        """.stripMargin
+
+      c.init(initPayload(code))._1 should be(200)
+
+      val payloads = Seq(JsObject("arg1" -> JsString("value1")), JsObject("arg2" -> JsString("value2")))
+
+      val responses = c.runMultiple(payloads.map {
+        runPayload(_)
+      })
+      payloads.foreach { a =>
+        //responses should contain(200, Some(JsObject("args" -> a)))
+      }
+    }
+
+    checkStreams(out, err, {
+      case (o, e) =>
+        o.replaceAll("\n", "") shouldBe "interleave meinterleave me"
```

Comment:
I don't see how are you testing that actions are actually running at the same time, and also not in different containers.